examples/adminapi.py

```diff
@@ -260,6 +261,37 @@ def example_delete_acls(a, args):
             raise
 
 
+def example_incremental_alter_configs(a, args):
+    """ Incrementally alter configs, keeping non-specified
+    configuration properties with their previous values.
+
+    Input Format : ResourceType1 ResourceName1 Key=Operation:Value;Key2=Operation2:Value2
+    ResourceType2 ResourceName2 ...
+
+    Example: TOPIC T1 compression.type=SET:lz4;cleanup.policy=ADD:compact TOPIC T2 compression.type=SET:gzip ...
+    """
+    resources = []
+    for restype, resname, configs in zip(args[0::3], args[1::3], args[2::3]):
+        incremental_configs = []
+        for name, operation_and_value in [conf.split('=') for conf in configs.split(';')]:
+            operation, value = operation_and_value.split(':')
```

Comment:
What is the expectation from the input for the delete case? The code gives error in the following case :-

`TOPIC T1 compression.type=DELETE`

I think we should check if `:` is there or not first. If not, then operation should be DELETE. If yes, then we can allow any operation (In case of DELETE, it would ignore value part)

Reply:
the idea was that you passed an empty string after the ":", but I can do this change